src/app/examples/toolbar-example/toolbar-example.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'kirby-toolbar-example',
+  templateUrl: './toolbar-example.component.html',
+  styleUrls: ['./toolbar-example.component.scss'],
+})
+export class ToolbarExampleComponent implements OnInit {
+  constructor() {}
+
+  ngOnInit() {}
+
+  backButtonSelected(): void {
+    alert('backbutton clicked!');
```

Comment:
Could you please do a back navigation here? Else i am trapped in the example and can never get back to the menu.